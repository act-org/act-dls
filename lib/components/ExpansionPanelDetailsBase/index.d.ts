/**
 * @prettier
 */
import * as React from 'react';
import { ExpansionPanelDetailsProps } from '@material-ui/core';
export declare type Props = ExpansionPanelDetailsProps;
declare const ExpansionPanelDetailsBase: React.FC<Props>;
export default ExpansionPanelDetailsBase;
//# sourceMappingURL=index.d.ts.map