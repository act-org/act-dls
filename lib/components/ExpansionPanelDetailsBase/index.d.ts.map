{"version":3,"file":"index.d.ts","sourceRoot":"","sources":["../../../src/components/ExpansionPanelDetailsBase/index.tsx"],"names":[],"mappings":"AAAA;;GAEG;AAEH,OAAO,KAAK,KAAK,MAAM,OAAO,CAAC;AAC/B,OAAO,EAEL,0BAA0B,EAC3B,MAAM,mBAAmB,CAAC;AAE3B,oBAAY,KAAK,GAAG,0BAA0B,CAAC;AAE/C,QAAA,MAAM,yBAAyB,EAAE,KAAK,CAAC,EAAE,CAAC,KAAK,CAEmB,CAAC;AAEnE,eAAe,yBAAyB,CAAC"}