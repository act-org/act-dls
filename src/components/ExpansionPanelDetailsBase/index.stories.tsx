/**
 * @prettier
 */

import * as React from 'react';

import ExpansionPanelDetailsBase from '.';

export const Base = (): React.ReactElement<any> => (
  <ExpansionPanelDetailsBase>
    <div>ExpansionPanelDetails</div>
  </ExpansionPanelDetailsBase>
);

export default {
  component: ExpansionPanelDetailsBase,
  parameters: {
    info: {
      text: 'This is the base variant of the ExpansionPanelDetails component.',
    },
  },
  title: 'Atoms/ExpansionPanelDetails',
};
