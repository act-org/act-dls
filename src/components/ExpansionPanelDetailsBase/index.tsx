/**
 * @prettier
 */

import * as React from 'react';
import {
  ExpansionPanelDetails,
  ExpansionPanelDetailsProps,
} from '@material-ui/core';

export type Props = ExpansionPanelDetailsProps;

const ExpansionPanelDetailsBase: React.FC<Props> = (
  props: Props,
): React.ReactElement<any> => <ExpansionPanelDetails {...props} />;

export default ExpansionPanelDetailsBase;
