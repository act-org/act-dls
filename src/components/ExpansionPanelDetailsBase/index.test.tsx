/**
 * @prettier
 */

import * as React from 'react';

import render from '~/helpers/test/render';

import ExpansionPanelDetailsBase from '.';

describe('ExpansionPanelDetailsBase', () => {
  it('matches the snapshot', () => {
    const { container } = render(
      <ExpansionPanelDetailsBase>
        <div>ExpansionPanelDetailsBase</div>
      </ExpansionPanelDetailsBase>,
    );

    expect(container).toMatchSnapshot();
  });
});
