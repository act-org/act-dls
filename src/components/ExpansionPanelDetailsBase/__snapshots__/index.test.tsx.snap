// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ExpansionPanelDetailsBase matches the snapshot 1`] = `
<div>
  <div
    class="MuiExpansionPanelDetails-root"
  >
    <div>
      ExpansionPanelDetailsBase
    </div>
  </div>
</div>
`;
